src/bin/hq.rs

```diff
@@ -281,12 +281,21 @@ fn make_global_settings(opts: CommonOpts) -> GlobalSettings {
     GlobalSettings::new(server_dir, color_policy)
 }
 
+fn set_colored_settings(settings: &GlobalSettings) {
```

Comment:
Maybe pattern match directly on ColorPolicy? (opts.colors?)

Reply:
I would keep it like this, you can in theory create global settings out of any input interface, and it's our own struct :)

Reply:
But ColorChoice is not our enum and your "_" may fail if they add something into it. But it is just an micro issue:)

Reply:
Oh, I see. Well, then I think `GlobalSettings` should contain `ColorPolicy` instead of `ColorChoice`, and `ColorChoice` should use `#[non_exhaustive]` :)

Anyway, thanks to `_` it will compile even if `ColorChoice` changes.

Reply:
I meant "fail" in semantics. But actually as we create the value ourselves so it is ok. However, you are right, in general, gsettings should probably hold our structure and create ColorChoice on-fly.